fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -0,0 +1,119 @@
+import { useLazyQuery } from '@apollo/react-hooks';
+import ApolloClient from 'apollo-client';
+import { FieldMessage } from 'braid-design-system';
+import { FieldProps } from 'braid-design-system/lib/components/private/Field/Field';
```

Comment:
I think the preferred pattern for Braid is to use something like `React.ComponentProps<typeof TextField>`